components/script/dom/bindings/callback.rs

```diff
@@ -15,7 +15,8 @@ use crate::dom::globalscope::GlobalScope;
 use crate::dom::window::Window;
 use js::jsapi::Heap;
 use js::jsapi::JSAutoRealm;
-use js::jsapi::{AddRawValueRoot, IsCallable, JSContext, JSObject};
+use js::jsapi::{AddRawValueRoot, IsCallable, /*JSContext, */JSObject};
+use crate::script_runtime::JSContext;
```

Comment:
@jdm and I also tried to re-import a new, safe JSContext. I am not sure if this helps since the compiler just spits out a lot of errors.

Reply:
Yeah, I would expect this to cause a lot of errors because now there are many functions accepting `*mut script_runtime::JSContext`. I would recommend undoing the changes to most of these files. The only time we should be changing code from `cx` to `*cx` is when `cx` is of type `script_runtime::JSContext` and the receiving code expects `*mut jsapi::JSContext`.